src/utils/helpers/browser/browser.js

```diff
@@ -89,6 +98,16 @@ const Browser = {
   },
 
   /**
+   * Sets the body scroll
+   *
+   * @param {Boolean} enabled
+   * @method setBodyScroll
+   */
+  setBodyScroll: (enabled) => {
+    Browser.getBody().style.overflow = enabled ? '' : 'hidden';
```

Comment:
I would just add it to where like this 

https://github.com/Sage/carbon/blob/master/src/utils/css/modules/utility-classes.scss

Then 

```
import { thing } from 'css path'
addClass(thing)
```

Something like that